lib/auth/auth_login_test.go

```diff
@@ -772,39 +780,27 @@ func TestServer_Authenticate_headless(t *testing.T) {
 		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
-			tc := tc
-			t.Parallel()
```

Comment:
Why was the parallel removed?

Reply:
I thought it improved the speed but on re-testing I was mistaken. I've added it back.